elements/pl-file-editor/pl-file-editor.py

```diff
@@ -12,8 +12,9 @@
 SOURCE_FILE_NAME_DEFAULT = None
 MIN_LINES_DEFAULT = None
 MAX_LINES_DEFAULT = None
-AUTO_RESIZE_DEFAULT = 'true'
+AUTO_RESIZE_DEFAULT = True
 PREVIEW_DEFAULT = None
+FOCUS_DEFAULT = True
```

Comment:
lets actually just set this to false 😀

Reply:
I will, need to edit the pending docs too